node/tcp/src/helpers/config.rs

```diff
@@ -73,12 +73,6 @@ impl Config {
 impl Default for Config {
     /// Initializes a new Tcp configuration with the default values.
     fn default() -> Self {
-        #[cfg(feature = "test")]
```

Comment:
Was this called out as unused by the compiler?

Reply:
That's right, the `tcp` crate doesn't have a `test` feature.